source/standard_scores.Rmd

```diff
@@ -0,0 +1,1515 @@
+points to estimate the quantile value for a QP that does not correspond to
+a whole-number rank.  $0000$
+
+:::{.callout-note}
+## Calculating quantiles using the linear method
+
+We gave a graphical explanation of how to calculate the quantile for a QP
+that does not correspond to whole-number rank in the data. Another explanation
+is to think of the quantile value as being a *weighted average* of the quantile
+values for the QPs of the whole number ranks just less than, and just greater
+than the QP we are interested in.  For example, let us return to the QP of
+$0.01$.  Let us remind ourselves of the QPs, whole-number ranks and
+corresponding values either side of the QP $0.01$:
+
+| Rank | Quantile position | Quantile value |
+|------|-------------------|----------------|
+| 5    | 0.0099            | 37933          |
+| 5.4  | 0.01              | **V**          |
+| 6    | 0.0113            | 40319          |
+
+: Ranks, QPs and corresponding values around QP of 0.01
+
+What value should we should give $V$ in the table?  One answer is to take the
+average of the two values either side of the desired QP — in this case $(37933
++ 40319) / 2$.  We could write this same calculation as $37933 * 0.5
++ 40319 * 0.5$ — showing that we are giving equal weight ($0.5$) to the two
+values either side.
+
+But giving both values equal weight doesn't seem quite right, because the QP we
+want is closer to the QP for rank 5 (and corresponding value 37933) than it is
+to the QP for rank 6 (and corresponding value 40319).  We should give more
+weight to the rank 5 value than the rank 6 value.   Specifically the lower
+value is 0.4 rank units away from the QP rank we want, and the higher is 0.6
+rank units away.  So we give higher weight for shorter distance, and multiply
+the rank 5 value by $1 - 0.4 = 0.6$, and the rank 6 value by $1 - 0.6 = 0.4$.
+Therefore the weighted average is $37933 * 0.6 + 40319 * 0.4 = 38887.4$. This
+is a mathematical way to get the value we described graphically, of tracking up
+from the rank of 5.4 to the line drawn between the values for rank 5 and 6, and
+reading off the y-value at which this track crosses that line.
+:::
+
+
+## Unusual values compared to the distribution
+
+Now we return the problem of whether KOMs and AOCs districts are unusual in
+terms of their median household incomes.  From what we have so far, we might
+conclude that AOC's district is fairly poor, and KOM's district is relatively
+wealthy.  But — are either of their districts *unusual* in their wealth or
+poverty?
+
+To answer that question, we have to think about the *distribution* of values.
+Are either AOC's or KOM's district outside the *typical spread* of values for
+districts?
+
+The rest of this section is an attempt to answer what we could mean by
+*outside* and *typical spread*.
+
+Let us start with a histogram of the district incomes, marking the position of
+the KOM and AOC districts.
```

Comment:
Explain. *....using 25 bins. The histogram shows the counts within each bin.*? Not sure how much we can assume of the students.